core/templates/pages/admin-page/admin-page.constants.ts

```diff
@@ -23,6 +23,10 @@ export const AdminPageConstants = {
   ADMIN_TOPICS_CSV_DOWNLOAD_HANDLER_URL: '/admintopicscsvdownloadhandler',
 
   ADMIN_JOB_OUTPUT_URL_TEMPLATE: '/adminjoboutput?job_id=<jobId>',
+  MEMORY_CACHE_HANDLER_URL: '/memorycacheadminhandler',
+  SEND_DUMMY_MAIL_HANDLER_URL: '/senddummymailtoadminhandler',
+  UPDATE_USERNAME_HANDLER_URL: '/updateusernamehandler',
+  NUMBER_OF_DELETION_REQUEST_HANDLER_URL:'/numberofdeletionrequestshandler',
```

Comment:
This is still not sorted

Reply:
@nithusha21 sending PR

Reply:
> This is still not sorted

@nithusha21 @DubeySandeep  while pushing,lint checker is stuck at third party checks,need some help

Reply:
> > This is still not sorted
> 
> @nithusha21 @DubeySandeep while pushing,lint checker is stuck at third party checks,need some help

@nithusha21 @DubeySandeep 
I tested and fixed lint test using command python -m scripts.linters.pre_commit_linter --file "filename"
There is something wrong with  "core/templates/pages/admin-page/admin-page.constants.ts"
It is neither running (stuck at third part lint checks) nor letting me push, everything else is fixed, can you please take a look?